code/modules/admin/verbs/randomverbs.dm

```diff
@@ -48,17 +48,18 @@
 	set category = null
 	set name = "Subtle Message"
 
-	if(!ismob(M))	return
-	if (!holder)
-		to_chat(src, "Only administrators may use this command.")
+	if(!ismob(M))	
+		return
+	if(!check_rights(R_ADMIN|R_MOD|R_MENTOR))
+		to_chat(src, "Only staff members may use this command.")
 		return
 
 	var/msg = input("Message:", text("Subtle PM to [M.key]")) as text
 
-	if (!msg)
+	if(!msg)
 		return
 	if(usr)
-		if (usr.client)
+		if(usr.client)
```

Comment:
This is the same as `if(usr && usr.client && usr.client.holder)` but can be shortened for better performance as `if(usr?.client?.holder)`